build/update-csv.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+YQ_MAJ_VERSION=${1:-4}
+installYQ() {
+    printf "Installing yq ..."
+    GO111MODULE=off go get -u github.com/myitcv/gobin
+    PATH=$(go env GOPATH)/bin:$PATH
+    gobin github.com/mikefarah/yq/v4@v4.6.1
+    yq --version
+}
+
+OPERATOR_ROLES_FILES=(
+    "deploy/role.yaml"
+    "deploy/clusterrole.yaml"
+)
+
+OPERATOR_CSV_FILE="deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml"
+
+#Ensure that GOPATH/bin is in the PATH
```

Comment:
```suggestion
PATH=$(go env GOPATH)/bin:$PATH
```

Reply:
@dmvolod I would change the user path only if yq is not available, i.e. move this inside the if statem

Reply:
In this case how to validate `yq` in the PATH if PATH is not set?
For me script always install yq even if it already installed

Reply:
if the user already have the correct yq in the path nothing is installed

Reply:
For my laptop GOPATH/bin never installed in the PATH and `yq` always reinstalling on every codegen
It's better to set validate it, set GOPATH/bin into the PATH and validate again